### DISCUSSION TITLE
Tanzu Community Edition strengthens VMware’s credibility as a key open-source player

### DISCUSSION THREAD

**Discussion by ppiutti** [vmware-tanzu/community-edition#2744] [Show and tell]
Hi all, I wrote something about the importance of TCE in relation to VMware's role in the Open Source landscape.
I hope you'll find it interesting:

https://blog.vgeek.it/2021/11/15/tanzu-community-edition-strengthens-vmwares-credibility-as-a-key-open-source-player/

**Comment by qnetter** [comment]
Awesome! Thank you!
